src/core/Entry.cpp

```diff
@@ -494,6 +494,18 @@ Entry* Entry::clone(CloneFlags flags) const
     entry->m_data = m_data;
     entry->m_attributes->copyDataFrom(m_attributes);
     entry->m_attachments->copyDataFrom(m_attachments);
+
+    if (flags & CloneUserAsRef) {
+        // Build the username refrence
+        QString username = "{REF:U@I:" + m_uuid.toHex() + "}";
+        entry->m_attributes->set(EntryAttributes::UserNameKey, username.toUpper(), m_attributes->isProtected(EntryAttributes::UserNameKey));
+    }
+
+    if (flags & ClonePassAsRef) {
+        QString password = "{REF:P@I:" + m_uuid.toHex() + "}";
```

Comment:
QString username = "{REF:U@I:" + m_uuid.toHex() + "}";
+        entry->m_attributes->set(EntryAttributes::UserNameKey, username.toUpper(), m_attributes->isProtected(EntryAttributes::UserNameKey));

Am i misinterpreting that, or do u simply uppercase the `username` at line 501? Just remove it?